support/client/lib/vwf/model/mil-sym.js

```diff
@@ -481,6 +535,14 @@ define( [ "module",
                     case "mobility":
                     case "taskForce":
                     case "installation":
+                    case "x":
+                    case "y":
+                    case "width":
+                    case "height":
+                    case "controlPts":
+                    case "visible":
+                    case "listening":
+                    case "symbolType":
```

Comment:
`visible` and `listening` don't have `settingProperty` cases, so they'll always return `undefined`, which is the same as not handling them here. That won't cause a problem, but they should probably be removed if this driver doesn't own them.